app/domain/authentication/authn_oidc/authenticate.rb

```diff
@@ -51,6 +52,14 @@ def new_token(input)
           username: input.username
         )
       end
+
+      def conjur_username(request_body)
+        id_token_username_field = "preferred_username"
+        conjur_username = request_body.id_token[id_token_username_field]
+        raise IdTokenFieldNotFound, id_token_username_field unless conjur_username&.present?
```

Comment:
I don't think you would ever need the safe navigation operator `&.` when using `present?`. The method is available on `nil`.

Reply:
correct. didn't know that.